e2e/ansible/roles/k8s-gke/tasks/main.yml

```diff
@@ -0,0 +1,26 @@
+---
+- name: Get the details of the available nodes
+  shell: kubectl get nodes | grep gke | awk {'print $1'}
+  args:
+    executable: /bin/bash
+  register: result_node
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+- name: Creating random name for the disks
+  shell: echo $(mktemp)| tr '[:upper:]' '[:lower:]' | cut -d '.' -f 2
+  args:
+    executable: /bin/bash
+  register: name
+  delegate_to: "{{groups['kubernetes-kubemasters'].0}}"
+
+- name: Creating the required number of disk 
+  shell: >
+    for ((i=0; i< "{{ disk_count }}"; i++));do
+      gcloud compute disks create {{ item }}-{{ name.stdout }}-${i}  --size "{{ disk_size }}" --type "{{ disk_type }}" --zone "{{ cluster_zone }}" &&
```

Comment:
Would recommend using --quiet flag at the end of these API calls.

Reply:
@yudaykiran  this --quite flag does not make any difference while calling this commands.